detox/ios/Detox/Actions/UIScrollView+DetoxActions.m

```diff
@@ -266,6 +266,8 @@ - (void)dtx_scrollWithOffset:(CGPoint)offset normalizedStartingPoint:(CGPoint)no
 
 - (void)_dtx_scrollWithOffset:(CGPoint)offset normalizedStartingPoint:(CGPoint)normalizedStartingPoint strict:(BOOL)strict
 {
+	[self dtx_assertVisibleWithPercent:@1];
+
```

Comment:
I don't like this.
However, I don't like the current approach also (>=75% visible), what we use in other views actions ([see for example](https://github.com/wix/Detox/blob/260e32a9b50f1a38f21bfbce7461147da8733625/detox/ios/Detox/Actions/UIPickerView%2BDetoxActions.m#L17)), since the view can be half visible and still accessible..

Also, the `normalizedStartingPoint` is undefined in many cases and then we use some almost-arbitrary point (from one of the edges). Theoretically I could use this point to check whether it's hittable on that point, but then it might result in breakage of tests (where this points are hidden but the view is partially visible, like in `03.actions-scroll.test.js`).

Reply:
It might be the best we can offer for this bug fix, of-course I'd love to hear more thoughts 😄

Reply:
I agree that adding the ≥75 assertion offhand would be reckless, and also will not make sense as long as the percentage cannot be specified by the user.

As for the idea you suggested - checking hittability, I wouldn't get it off the table altogether. In any case, on Android we essentially do something quite similar with respect to visibility -- asserting the view is non-empty.

Reply:
Done ✔️ we will now check hittability instead of visibility.
There's a lot of changes in the new implementation, so this may be effective to review them face to face.

Reply:
of course, we should also test it with the latest changes as broadly as possible using the internal wix projects. especially when it adds more assertions to touch actions.